source/Octopus.Client/Repositories/ReleaseRepository.cs

```diff
@@ -70,5 +71,10 @@ public LifecycleProgressionResource GetProgression(ReleaseResource release)
         {
             return Client.Get<LifecycleProgressionResource>(release.Links["Progression"]);
         }
+
+        public ResourceCollection<ReleaseLogResource> GetLog(ReleaseResource release, int skip = 0, int? take = null)
+        {
```

Comment:
This should also have the check for the "Log" link and the message about the required server version

Reply:
Okay done.